example/lib/model/ble_peripheral.dart

```diff
@@ -6,8 +6,20 @@ class BlePeripheral extends Equatable {
   final int rssi;
   final bool isConnected;
 
+  BlePeripheralCategory get category => _nameToCategory(name);
+
   BlePeripheral(this.name, this.id, this.rssi, this.isConnected);
 
+  BlePeripheralCategory _nameToCategory(String name) {
+    if (name == "SensorTag") {
```

Comment:
could you introduce enum or at least static in order to not use raw string

Reply:
Added `KnownBlePeripheralCategory` helper class. Created an issue to determine peripheral's category in a more sophisticated way (not only based on it's name)